.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,2073 @@
+name: "Build & Test"
+
+on:
+  push:
+    branches:
+      - master
+      - branch-DDS-3.15
+      - branch-DDS-3.14
+      - branch-DDS-3.13
```

Comment:
Does it allow wildcard expressions?  We don't necessarily need to do it now but maybe next iteration of this script could use them.